charts/lunar-mcpx-webapp/templates/lunar-hive-ui-deployment.yaml

```
@@ -0,0 +1,89 @@
+{{- if .Values.ui.enabled | default true }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "lunar-mcpx-webapp.fullname" . }}-ui
+  labels:
+    name: {{ include "lunar-mcpx-webapp.fullname" . }}-ui
+    app: {{ include "lunar-mcpx-webapp.fullname" . }}-ui
+spec:
+  replicas: {{ .Values.ui.replicaCount | default 1 }}
+  selector:
+    matchLabels:
+      app: {{ include "lunar-mcpx-webapp.fullname" . }}-ui
+      {{- include "lunar-mcpx-webapp.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "lunar-mcpx-webapp.fullname" . }}-ui
+        {{- include "lunar-mcpx-webapp.selectorLabels" . | nindent 8 }}
+    spec:
+      containers:
+        - name: mcpx-ui-container
+          image: "{{ .Values.ui.image.repository }}:{{ .Values.ui.image.tag | default .Values.global.appVersion | default .Chart.AppVersion }}"
+          ports:
+            - containerPort: 5173
+          resources:
+            {{- toYaml .Values.ui.resources | nindent 12 }}
+          envFrom:
+            - secretRef:
+                name: {{ include "lunar-mcpx-webapp.fullname" . }}-embedded
+          {{- with .Values.global.extraEnvFromSecrets }}
+          {{- range $.Values.global.extraEnvFromSecrets }}
+            - secretRef:
+                name: {{ . }}
+          {{- end }}
+          {{- end }}
+          {{- with .Values.ui.extraEnvFromSecrets }}
+          {{- range $.Values.ui.extraEnvFromSecrets }}
+            - secretRef:
+                name: {{ . }}
+          {{- end }}
+          {{- end }}
+          env:
+            - name: VITE_ENABLE_ENTERPRISE
+              value: "true"
+            - name: VITE_MCPX_SERVER_URL
+              value: http://{{ include "lunar-mcpx-webapp.fullname" . }}-router
```

```suggestion
              value: https://{{ first .Values.ingress.domains.router }}
```

As this is being called from the user browser we need to make it access the ingress